Face/Verify/Program.cs

```diff
@@ -0,0 +1,289 @@
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName2}`.");
+                    return;
+                }
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName2}`.");
+                if (detectedFaces[0].FaceId == null)
+                {
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for verification purpose.");
+                    return;
+                }
+
+                sourceFaceId2 = detectedFaces[0].FaceId.Value;
+            }
+
+
+            // Verification example for faces of the same person.
+            VerifyResult verifyResult1 = await Client.Face.VerifyFaceToFaceAsync(sourceFaceId1, targetFaceIds[0]);
+            Console.WriteLine(
+                verifyResult1.IsIdentical
+                    ? $"Faces from {SourceImageFileName1} & {TargetImageFileNames[0]} are of the same (Positive) person, similarity confidence: {verifyResult1.Confidence}."
+                    : $"Faces from {SourceImageFileName1} & {TargetImageFileNames[0]} are of different (Negative) persons, similarity confidence: {verifyResult1.Confidence}.");
+
+            // Verification example for faces of different persons.
+            VerifyResult verifyResult2 = await Client.Face.VerifyFaceToFaceAsync(sourceFaceId2, targetFaceIds[0]);
+            Console.WriteLine(
+                verifyResult2.IsIdentical
+                    ? $"Faces from {SourceImageFileName2} & {TargetImageFileNames[0]} are of the same (Negative) person, similarity confidence: {verifyResult2.Confidence}."
+                    : $"Faces from {SourceImageFileName2} & {TargetImageFileNames[0]} are of different (Positive) persons, similarity confidence: {verifyResult2.Confidence}.");
+        }
+
+        public static async Task Verify_in_PersonGroup()
+        {
+            // Create a person group.
+            string GroupId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create a person group ({GroupId}) {Environment.NewLine}");
+            await Client.PersonGroup.CreateAsync(GroupId, GroupId);
+
+            // Create a person group person.
+            Person p = new Person();
+            p.Name = "Dad";
+            p.UserData = "Person for sample";
+            Console.WriteLine($"Create a person group person '{p.Name}' {Environment.NewLine}");
+            p.PersonId = (await Client.PersonGroupPerson.CreateAsync(GroupId, p.Name)).PersonId;
+
+            foreach (var targetImageFileName in TargetImageFileNames)
+            {
+                // Read image file. 
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Add face to the person group. 
+                    Console.WriteLine($"Add faces to the person group from image `{targetImageFileName}`.");
```

Comment:
`Console.WriteLine($"Add face to the large person group person from image `{targetImageFileName}`.");`

Same for other places.

Reply:
done